docs/debugConfig.md

```diff
@@ -0,0 +1,22 @@
+# Breaking changes to JavaScript Debug Configuration
```

Comment:
should this mention JavaScript?

Reply:
Yeah I think so. C# wasn't affected. Python isn't supported by this extension yet. And these instructions don't work for Java (The partner team is handling Java support and I think they fixed everything directly in Maven. In any case, we can always change this after the fact to include Java since the aka.ms link will point directly to master)